---
layout: post
title: How Renseq technology will save canola from clubroot
tags: 
- Bioinformatics
- Research
author: Jiaxu Wu
date: 2023-04-02 15:32 +0800
---
**Resistance gene enrichment sequencing (RenSeq)** is an NLR gene-targeted, enrichment and sequencing method that allows us to discover 
and annotate the full complement of genes that encode NLR genes (Steuernagel et al. 2016).  

  
Several tools have also been developed to annotate plant NLR genes in a rapid way, such as NLR-parser (Steuernagel et al. 2015), NLGenomeSweeper (Toda et al. 2020) and NLR-Annotator 
(Steuernagel et al. 2020). Ren-seq allows us to design bait libraries using NLR sequences predicted from plant genomes. After that, the complementarity of 
isolated DNA to baits can enrich NLR encoding fragments, and more than 500-fold enrichment of NLR genes relative to other genes can be achieved. 
The repetitive nature of NLR genes hampered de novo assembly of this family. Reads generated are around 3-4 kb in length (longer than average NLR sequence). 
These long sequence reads are especially well suited for de novo assembly of whole NLR genes including their regulatory elements. Ren-Seq has been 
successfully employed in Solanum americanum, tomato, wheat, and potato (Andolfo et al. 2014; Arora et al. 2023; Steuernagel et al. 2016; Witek et al. 2016a; 
Witek et al. 2021; Zhang et al. 2021a). Recently, Zhang et al. (2021a) successfully isolated two, unrelated stem rust resistance NLR genes, Sr26 and Sr61, 
from Thinopyrum ponticum by employing MutRenSeq (Mutagenesis RenSeq).  
It has not yet been not possible to apply RenSeq in *B. napus* due to the low-quality reference genome, a limited understanding of resistance genes in this species, and restricted access to disease-resistant canola germplasm. 
Recent advances such as the resequencing of canola and other Brassica spp. whole genomes (Chen et al. 2021b; Song et al. 2020; Yang et al. 2022), 
as well as the characterization of disease-resistance genes (Dolatabadian et al. 2020), offer an ideal opportunity to design the NLR baits 
required for RenSeq. Overall, RenSeq could be applied to Brassica crops to investigate NLR-mediated clubroot resistance, but this should happen in 
close collaboration with the industry to finally understand the mechanisms behind CR cultivars used by growers and move towards the development of 
new and more stable elite cultivars.  
In this project, we combine RenSeq (a state-of-the-art sequencing method) and confirmatory analysis (i.e., transformation and proteomics) exclusively using commercial 
canola germplasm. The NLR genes responsible for CRM, CR1 and CRB resistance will be identified and characterized to provide a toolbox that strengthens current CR canola breeding programs. Understanding the resistance mechanisms will allow researchers to make informed breeding decisions and increase the stability and success of varieties combining these resistance profiles.

