---
layout: post
title: Clubroot Detection
author: Jiaxu Wu
tags:
- Plant biology
- Research
toc:  false
date: 2023-11-05 12:00 -0400
---

Efficient disease management starts with a reliable and sensitive diagnostic method to stop the spreading of the pathogen and its economic losses.  


Although this area has also been very challenging, in the last decade, several molecular methods have been developed to detect *P. brassicae* resting spores in infected fields, if possible, before typical symptoms can be observed. In addition, several PCR-based assays have been developed in the last ten years targeting diverse genes aiming the detection of the pathogen in infected fields, quantification of the infection load, or the differentiation of P. brassicae pathotypes with different virulence profiles.  
The nuclear ribosomal DNA (rDNA), comprising the small subunits (SSUs), the large subunit (LSU), the internal transcribed spacers (ITS1 and ITS2), and the 5.8S, has been extensively used as a marker to identify *P. brassicae* and to differentiate geographical isolates of the pathogen . There is evidence that supports that single nucleotide polymorphisms, insertions, and deletions in these regions can differentiate *P. brassicae* isolates based on geographic origin but also based on their virulence, like the case of the highly virulent pathotype 5, now Pb5X identified in Canada (Table 1). These markers have also been extensively used to develop quantitative PCR assays to study the distribution of the clubroot pathogen in the fields and other epidemiological aspects of the disease (Table 1).  
With the discovery of new and diverse pathotypes, the challenge has been identifying areas of the genome useful to differentiate virulent from avirulent pathotypes in combination with more resolutive methods. An example of that is the use of P. brassicae cpn60 universal target (cpn60UT) to develop digital droplet PCR (ddPCR) and loop-mediated isothermal DNA amplification (LAMP), proving to be a suitable market to study the distribution of the pathogen in infected fields and to differentiate pathotypes (Table 1). Another method extensively used to differentiate clubroot pathotypes is RNase H-dependent PCR (rhPCR), amplifying rDNA or fragments of the genome unique for different pathotypes identified through comparative genomics (Table 1).  
Visible/near-infrared hyperspectral (HSI) imaging with convolutional neural networks (CNN) is another technique used to detect clubroot by estimating the plant physiological abnormalities induced by the clubroot pathogen. However, this technique has considerable limitations because it only detects the disease after the successful infection has been established and the plant shows visible symptoms, the resting spores are already spread through the field (Table 1).   

| Method      | Description |
| :---        |    :----:   |
| PCR         |Detection of *P. brassicae*, differentiation of *P. brassicae* pathotypes based on virulence or geographic distribution|
| qPCR        |Study of *P. brassicae* distribution and quantification of spore number in the field|
|ddPCR        |Quantification of *P. brassicae* and study of the pathogen dynamics in the field|
|rhPCR        |Study of *P. brassicae* population polymorphisms and differentiation of clubroot pathotypes with differential virulence profiles|
|LAMP         |Fast and sensitive detection of *P. brassicae*|
|HSI+         |Detection of clubroot disease without destroying the plants infected|
|CNN          |Same as HSI+|  

**Reference**  
Javed, M.A., Schwelm, A., Zamani-Noor, N., Salih, R., Silvestre Vañó, M., Wu, J. et al. (2023) The clubroot pathogen Plasmodiophora brassicae: A profile update. Molecular Plant Pathology, 24, 89–106. Available from: https://doi.org/10.1111/mpp.13283  



